src/D2L.Bmx/Program.cs

```diff
@@ -106,32 +134,30 @@
 		role: context.ParseResult.GetValueForOption( roleOption ),
 		duration: context.ParseResult.GetValueForOption( durationOption ),
 		nonInteractive: context.ParseResult.GetValueForOption( nonInteractiveOption ),
-		output: context.ParseResult.GetValueForOption( formatOption )
+		format: context.ParseResult.GetValueForOption( formatOption )
 	);
-} ) );
-
-rootCommand.Add( printCommand );
+} );
 
 // bmx write
 var outputOption = new Option<string>(
 	name: "--output",
-	description: "write to the specified file path instead of ~/.aws/credentials" );
+	description: ParameterDescriptions.Output );
 var profileOption = new Option<string>(
 	name: "--profile",
-	description: "aws profile name" );
+	description: ParameterDescriptions.Profile );
 
-var writeCommand = new Command( "write", "Write to AWS credentials file" ) {
+var writeCommand = new Command( "write", "Write AWS credentials to the credentials file" ) {
 	accountOption,
+	roleOption,
 	durationOption,
 	orgOption,
 	outputOption,
 	profileOption,
-	roleOption,
 	userOption,
```

Comment:
The order here looks off.
Profile should be higher up. Org should be together with user.
I'd say: profile, duration, output, org, user

Reply:
Hmm yea it is off compared to the others where org and user are close. I've adjusted to your suggested order